### DISCUSSION TITLE
Not Creating Users?

### DISCUSSION THREAD

**Discussion by s3lverd** [DSJAS/DSJAS#52] [Installation help] [answered]
Hi! So im trying to create a user for logging into the bank.. but it says its successful but never seems to add it to database/website list. I was wondering if anyone could help. Thanks :)
![notthere](https://user-images.githubusercontent.com/101513517/183220194-7d54dc08-3f69-4c21-b1fb-5c776cf3c4e1.png)
![created](https://user-images.githubusercontent.com/101513517/183220197-d21e9e76-5420-4bfb-9774-40f790b3519c.png)

**Comment by s3lverd** [comment, marked as answer]
Nevermind I think I fixed it

**Reply by ejv2** [reply]
Great! For reference of other users, would you mind sharing what the problem was and how you solved it?

**Reply by s3lverd** [reply]
What I think fixed it was deleting the users table or whatever its called in the sql database and then resetting DSJAS. I don't know if all these steps actually helped but that's what seemed to work for me :)
